### ISSUE TITLE
[Bug]: Validation error with broker_client.update_account

### ISSUE THREAD

**Issue by xuru** [alpacahq/alpaca-py#164]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When updating the contact information of an account using the broker api, I get a ValidationError for Disclosures:
```
contact_data = UpdatableContact(email="test.carlos01@bob.us", phone_number="+14417491111", street_address=["1739 nowhere st"], city="bobsville", state="TX", postal_code="78660")
update_request = UpdateAccountRequest(contact=contact_data)
broker_client.update_account(account_id="1da21098-1cad-46e0-8718-ccc458b5490a", update_data=update_request)
Traceback (most recent call last):
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/IPython/core/interactiveshell.py", line 3398, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-95-069ba73dfa05>", line 1, in <cell line: 1>
    broker_client.update_account(account_id="1da21098-1cad-46e0-8718-ccc458b5490a", update_data=update_request)
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/alpaca/broker/client.py", line 224, in update_account
    return Account(**response)
  File "/Users/eric/.virtualenvs/wedge-backend-ghk8MMIr-py3.10/lib/python3.10/site-packages/alpaca/broker/models/accounts.py", line 269, in __init__
    parse_obj_as(Disclosures, response["disclosures"])
  File "pydantic/tools.py", line 38, in pydantic.tools.parse_obj_as
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ParsingModel[Disclosures]
__root__ -> employment_status
  value is not a valid enumeration member; permitted: 'unemployed', 'employed', 'student', 'retired' (type=type_error.enum; enum_values=[<EmploymentStatus.UNEMPLOYED: 'unemployed'>, <EmploymentStatus.EMPLOYED: 'employed'>, <EmploymentStatus.STUDENT: 'student'>, <EmploymentStatus.RETIRED: 'retired'>])
```
This works if I directly use `requests`.

### Expected Behavior

Be able to update the contact on an account.

### SDK Version I encountered this issue in

alpaca-py version 0.6.0

### Steps To Reproduce

```markdown
1. Add an account to sandbox or use an existing one that has a disclosure
2. Use the library to update only the contact information.
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

**Comment by forLinDre** [comment]
I get the same error using TradingClient's close_all_positions method.

**Comment by haxdds** [comment]
Fixed by #235
